### PR TITLE
RHDEVDOCS-6136: Added missing APIversion and Kind info

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

GitOps 1.13, GitOps 1.14

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-6136

Link to docs preview:

Step 4 of the section [Running the Argo CD instance at the cluster-level](https://79913--ocpdocs-pr.netlify.app/openshift-gitops/latest/declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.html#run-argo-cd-instance-on-cluster_configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations)

QE review:
- [x] QE has approved this change.

SME review: @jgwest 
QE review: @varshab1210 
Peer review: @GroceryBoyJr 

Additional information:

### CODE REVIEW

**Review by jgwest — APPROVED** [review]
LGTM, short and sweet :smile:

**Comment by varshab1210** [comment]
LGTM, thanks

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Review by skrthomas — COMMENTED** [review]
One small nit, lgtm otherwise.

File: modules/go-run-argo-cd-instance-on-infrastructure-nodes.adoc [hunk 1]
```diff
@@ -35,10 +35,16 @@ metadata:
 spec:
   runOnInfra: true
 ----
-. Optional: If taints have been added to the nodes, then add `tolerations` to the `GitOpsService` custom resource, for example:
+. Optional: If taints have been added to the nodes, then add `tolerations` to the `GitOpsService` custom resource.
++
+.Example
 +
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply modules/go-run-argo-cd-instance-on-infrastructure-nodes.adoc:40]: Fixed.

**Comment by Dhruv-Soni11** [comment]
Hi @skrthomas - I have fixed your review comments and squashed all the commits. PTAL and let me know if this can be merged. Thanks.

**Pull request merged**